### ISSUE TITLE
Enter animations don't work with CSS

### ISSUE THREAD

**Issue by ahmedu007** [tamagui/tamagui#2660]
### Current Behavior

Using `createAnimations` from `@tamagui/animations-css` don't trigger enter animations on unmounted components, such as dialog, toast etc. 

### Expected Behavior

They should work how they work with `createAnimations` from `react-native-reanimated`

### Tamagui Version

```markdown
"tamagui": "^1.98.0"
```

### Platform (Web, iOS, Android)

```markdown
Web. NextJS App router
```

### Reproduction

```markdown
Run: 
`npm create tamagui@latest`

In the `Animation.ts` file replace


// import { createAnimations } from '@tamagui/animations-react-native'
import { createAnimations } from '@tamagui/animations-css'
```
```


### System Info

```markdown
System:
    OS: macOS 14.4.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 2.51 GB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.11.1 - ~/.nvm/versions/node/v20.11.1/bin/node
    Yarn: 1.22.21 - /usr/local/bin/yarn
    npm: 10.2.4 - ~/.nvm/versions/node/v20.11.1/bin/npm
    bun: 1.1.6 - /usr/local/bin/bun
  Browsers:
    Chrome: 124.0.6367.209
    Edge: 125.0.2535.51
    Safari: 17.4.1
  npmPackages:
    @axe-core/playwright: ^4.8.5 => 4.9.0 
    @contentful/rich-text-react-renderer: ^15.19.6 => 15.19.6 
    @next/bundle-analyzer: ^14.1.4 => 14.2.3 
    @playwright/test: ^1.42.1 => 1.44.0 
    @reduxjs/toolkit: ^2.2.2 => 2.2.5 
    @tamagui/animations-css: ^1.94.5 => 1.98.0 
    @tamagui/config: ^1.94.5 => 1.98.0 
    @tamagui/next-plugin: ^1.94.5 => 1.98.0 
    @tamagui/toast: ^1.98.0 => 1.98.0 
    @testing-library/jest-dom: ^6.4.2 => 6.4.5 
    @testing-library/react: ^14.2.2 => 14.3.1 
    @testing-library/user-event: ^14.5.2 => 14.5.2 
    @types/jest: ^29.5.12 => 29.5.12 
    @types/node: ^20.11.30 => 20.12.12 
    @types/react: ^18.2.0 => 18.3.2 
    @types/react-dom: ^18.2.0 => 18.3.0 
    @types/whatwg-fetch: ^0.0.33 => 0.0.33 
    @typescript-eslint/eslint-plugin: ^7.6.0 => 7.9.0 
    contentful: ^10.8.7 => 10.11.0 
    eslint: ^8.57.0 => 8.57.0 
    eslint-config-next: ^14.1.4 => 14.2.3 
    eslint-config-prettier: ^9.1.0 => 9.1.0 
    eslint-plugin-jest: ^27.9.0 => 27.9.0 
    husky: ^9.0.11 => 9.0.11 
    jest: ^29.7.0 => 29.7.0 
    jest-environment-jsdom: ^29.7.0 => 29.7.0 
    jest-junit: ^16.0.0 => 16.0.0 
    lint-staged: ^15.2.2 => 15.2.2 
    msw: ^2.2.13 => 2.3.0 
    next: ^14.1.4 => 14.2.3 
    prettier: ^3.2.5 => 3.2.5 
    react: 18.2.0 => 18.2.0 
    react-dom: 18.2.0 => 18.2.0 
    react-native-svg: ^15.1.0 => 15.3.0 
    react-native-svg-web: ^1.0.9 => 1.0.9 
    react-native-web: ^0.19.10 => 0.19.11 
    react-redux: ^9.1.0 => 9.1.2 
    sharp: ^0.33.3 => 0.33.4 
    tamagui: ^1.98.0 => 1.98.0 
    typescript: ^5.4.3 => 5.4.5 
    undici: ^6.12.0 => 6.17.0 
    whatwg-fetch: ^3.6.20 => 3.6.20
```

**Comment by vitorqueirosz** [comment]
i'm having the same issue

**Comment by natew** [comment]
Still happening? We made some fixes here.

**Comment by ahmedu007** [comment]
It is still happening on version `^1.101.7`. I'll bump up deps and report back if issue is resolved.

**Comment by ahmedu007** [comment]
@natew still an issue on `1.103.0`. There are quite a few errors about displayName, but that something we can tackle when we actually bump it up 😅

<img width="722" alt="image" src="https://github.com/user-attachments/assets/8eaaae62-7f14-4293-a138-8a9af3232544">

**Comment by ahmedu007** [comment]
@vitorqueirosz if you want a work around for dialogs, you can do this in the meantime:

In the CSS file, you can target dialogs like: 
```
dialog[data-state='open'] {
  -webkit-animation: scale-up-top 0.3s cubic-bezier(0.39, 0.575, 0.565, 1) both;
  animation: scale-up-top 0.3s cubic-bezier(0.39, 0.575, 0.565, 1) both;
}

// add your keyframe here
```

**Comment by natew** [comment]
Going to be looking at this one today

**Comment by natew** [comment]
fixed in v1.103.3
